tests/dataprep/test_dataprep_multimodal_redis_langchain.sh

```diff
@@ -115,9 +120,17 @@ place to watch it is on BlackmagicShine.com. We're right here on the smoking
 00:00:45.240 --> 00:00:47.440
 tire.""" > ${transcript_fn}
 
+    echo "This is an apple."  > ${caption_fn}
+
+    echo "Downloading Image"
+    wget https://github.com/docarray/docarray/blob/main/tests/toydata/image-data/apple.png?raw=true -O ${image_fn}
```

Comment:
If we store all downloaded files into a clean `tmp_data` folder or something like, we can then remove that folder at the end.
Otherwise we have to keep track of each file being downloaded to remove them one by one prior to finishing the test.